pkg/config/resource.go

```diff
@@ -62,7 +62,19 @@ type ResourceConfig struct {
 	// these APIs, we basically need to revert to custom code because there's
 	// very little consistency to the APIs that we can use to instruct the code
 	// generator :(
-	UpdateOperation *UpdateOperationConfig `json:"update_operation,omitempty"`
+	UpdateOperation *CustomOperationsConfig `json:"update_operation,omitempty"`
+	// FindOperation contains instructions for the code generator to generate
+	// Go code for the find operation for the resource. For some resources,
+	// there is no describe/find/list apis. However, it is possible to write
+	// `find` function for such resources using another resource's (say,
+	// resource B) `find` function. Then, one could write custom code to find
+	// the relevant resource from the response of resource B's find function.
+	//
+	// Additionally, if there the `find` function for a resource is not
+	// available then reconciler's progression is stalled with the error
+	// `find` operation `notImplemented`. Custom find function allows the as
+	// reconciler's progression.
+	FindOperation *CustomOperationsConfig `json:"find_operation,omitempty"`
```

Comment:
To keep consistent naming across all the packages, let's name this `ReadOperation` or `ReadOneOperation` ?

Reply:
done